content/articles/how-to-ground-truth-label-your-datasets/index.md

```diff
@@ -0,0 +1,163 @@
+
+Also, you input how long you want to track it. It is specified beside the play section where you input the `start time` and `end time`. 
+
+For example, we are going to track the cars for seven seconds.
+
+![tracking](/engineering-education/how-to-ground-truth-label-your-datasets/ground12.png)
+
+After this, you click on `automate`. Once you click this, the window displays the instruction for using this algorithm. Read through these instructions and then click on the play at the top of the window.
+
+![automation](/engineering-education/how-to-ground-truth-label-your-datasets/ground12a.png)
+
+Once it completes, click on the `accept`. You can look at the `label summary` to see how the labeling proceeded.
+
+![label summary](/engineering-education/how-to-ground-truth-label-your-datasets/ground13.png)
+
+### Other capabilities
+Let's say that we want to label the lanes. In this case, we cannot use bounding boxes; we will use the line and label the lanes. It means that you draw lines on the lanes, and they will be labeled.
+
+![labeling the lane](/engineering-education/how-to-ground-truth-label-your-datasets/ground14.png)
+
+Also, you can label individual pixels in the image using the `pixel label`. For example; we want to label the road, so we will make a label road and use the pixel label.
+
+![using pixel label](/engineering-education/how-to-ground-truth-label-your-datasets/ground15.png)
+
+Note that you can erase sections that you never intended to be labeled using `erase`, and they also include those that you may have missed using the `brush`. We also have a feature, `smart polygon`. This feature allows you to make a surrounding of what you want to label and makes its segmentation. After all these, you can save your work.
+
+### Integrate ground truth information from other sensors.
+This feature allows you to integrate ground truth information from other signals. For example, let's open a video that has been time-synchronized with lidar data. 
+
+To open this, execute the command below:
+
+```Matlab
+groundTruthLabeler('01_city_c2s_fcw_10s.mp4', 'ConnectorTargetHandle', @LidarDisplay);
+```
+
+This command opens up two windows.
+
+![lidar windows](/engineering-education/how-to-ground-truth-label-your-datasets/ground16.png)
+
+The first is the ground truth labeler with the video data and the point cloud player, which contains synchronized lidar data. You notice that it is synchronized because if you scrub through the lidar player, we are also scrubbing through the video player. 
+
+This is important because lidar data can provide information that vision sensors alone cannot determine. For example, the lidar data provides the ego vehicle's distance from the surrounding objects.
+
+### Conclusion
+Ground truth labeler is so effective in the automation of the labeling process. Despite the labeling process that can be time-consuming and tedious, this App can do that with a button click. 
```

Comment:
Upload the video used in this tutorial as a drive link and paste it here. It will help readers follow the tutorial better.